web/src/settings_account.js

```diff
@@ -456,10 +457,13 @@ export function set_up() {
             password_quality = (await import("./password_quality")).password_quality;
             $("#pw_strength .bar").removeClass("hide");
 
-            $("#new_password").on("input", () => {
-                const $field = $("#new_password");
-                password_quality($field.val(), $("#pw_strength .bar"), $field);
-            });
+            const $field = $("#new_password");
+            $("#new_password").on(
```

Comment:
This can use the variable defined above.

Reply:
Oh, fixed it.